### DISCUSSION TITLE
Does skywalking java agent v8.4.0 not support plugin:apm-elasticsearch-7.x-plugin-8.4.0.jar?

### DISCUSSION THREAD

**Discussion by gaiwey** [apache/skywalking#10458] [Q&A]
The default supported plugins are only apm-elasticsearch-5.x-plugin-8.4.0.jar and apm-elasticsearch-6.x-plugin-8.4.0.jar, where to download apm-elasticsearch-7.x-plugin -8.4.0.jar?

**Comment by wu-sheng** [comment]
Check the latest support list, https://skywalking.apache.org/docs/skywalking-java/v8.14.0/en/setup/service-agent/java-agent/supported-list/

And why 8.4.0? It was a 2 years ago release.

**Reply by gaiwey** [reply]
The version used by the company

**Reply by wu-sheng** [reply]
So what? That is your env issue. Not community.

Try to learn how to talk with the upstream.
